src/Form/FormGroup.tsx

```diff
@@ -30,7 +29,7 @@ function FormGroup<As extends React.ElementType = 'div'>({
   size,
   as,
   ...props
-}: Props<As> & React.ComponentPropsWithoutRef<As>) {
+}: Props & { as?: As } & React.ComponentPropsWithoutRef<As>) {
```

Comment:
```suggestion
}: { as?: As } & Props & React.ComponentPropsWithoutRef<As>) {
```
Ah, sorry - it needs to be this way apparently. With that change, it looks like:

<img width="300" height="70" alt="Screenshot 2025-10-06 at 4 19 20 PM" src="https://github.com/user-attachments/assets/c3accf25-46fb-425c-a85b-be24c298c721" />

Reply:
Done! 👍